src/Lumper.Lib/BSP/Lumps/BspLumps/PakFileLump.cs

```diff
@@ -38,6 +39,30 @@ public class PakfileLump(BspFile parent) : ManagedLump<BspLumpType>(parent), IFi
 
     private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
 
+    //as per https://github.com/ValveSoftware/source-sdk-2013/blob/master/mp/src/utils/vbsp/cubemap.cpp
```

Comment:
This doesn't seem right. Maps I'm looking at often have hundreds of cubemaps stored in `maps/<name>/c_<x coord>_<y coord>_<z coord>.vtf`, these should also be moved. grepping for "maps/" in that CPP file seems to find what I'm talking about: https://github.com/ValveSoftware/source-sdk-2013/blob/0d8dceea4310fde5706b3ce1c70609d72a38efdf/mp/src/utils/vbsp/cubemap.cpp#L512